### DISCUSSION TITLE
Cannot see energy flow dots

### DISCUSSION THREAD

**Discussion by Tangiers2020** [slipx06/sunsynk-power-flow-card#518] [General]
Hi, unless i have missed something simple, i cannot see any energy indicator dots moving along the line.
Screenshots tell me it's a thing but cannot find anything in the documentation.

**Comment by slipx06** [comment]
Hi. Can you share your config. The animations are enabled by default and uses the `max_power:` and `animation_speed:`  attributes for each of the sections. They have default values assigned for it to work.
